package.json

```diff
@@ -11,7 +11,9 @@
     "test:debug": "echo '/*' && echo '  Navigate to about:inspect in Chrome' && echo '*/' && node --inspect node_modules/.bin/jest --runInBand",
     "test:ci": "CI=true jest --verbose=false --config ./jest.config.js",
     "test:watch": "jest --watch --verbose=false --config ./jest.config.js",
-    "start": "sam local start-api --port 5000",
+    "test:integration": "npm run build && jest --verbose=false --config ./jest.config.integration.js",
+    "test:watch:integration": "npm run build && jest --watch --verbose=false --config ./jest.config.integration.js",
+    "start": "sam local start-api --port 5000 2>&1 | tr \"\\r\" \"\\n\"",
```

Comment:
Added this part `2>&1 | tr \"\\r\" \"\\n\"` because the console.logs from inside the function were not shown. A workaround, but couldn't find anything better - https://github.com/aws/aws-sam-cli/issues/1359.